bert/src/config.rs

```diff
@@ -37,27 +37,16 @@ use crate::{
 /// ```toml
 /// # the config file is always named `config.toml`
 ///
-/// # optional, eg to enable the japanese pre-tokenizer
-/// [pre-tokenizer]
-/// path = "mecab"
-///
-/// # the path is always `vocab.txt`
+/// # the path is always `tokenizer.json`
 /// [tokenizer]
-/// cleanse-accents = true
-/// cleanse-text = true
-/// lower-case = false
-/// max-chars = 100
+/// add_special_tokens = true
```

Comment:
🔧 this contradicts the implementation where `-` are used in the name of this option instead of `_`.

either way is fine for me, we should just apply the casing consistently to all config option names.